docs/data/routes/docs/techniques/ssr/sitecore-experience-edge/en.md

```diff
@@ -0,0 +1,74 @@
+---
+name: ssr-experience-edge
+routeTemplate: ./data/component-templates/article.yml
+title: SSR using Sitecore Experience Edge
+---
+
+# Server-side rendering using Sitecore Experience Edge
+
+You can configure server-side rendering (SSR) for your JSS application using a Sitecore Experience Edge endpoint. 
+
+Sitecore provides a [sample for server-side rendering using a Sitecore Experience Edge endpoint](https://github.com/Sitecore/jss/tree/dev/samples/node-headless-ssr-experience-edge), Node.js, and Express. It renders your JSS application, querying layout and dictionary data, without proxying the requests.
+
+> The sample for SSR using a Sitecore Experience Edge endpoint is experimental and not officially supported by Sitecore.
```

Comment:
I think there may be some other places where we list experimental samples; might be a good idea to search the docs directory for "experimental" to check.

Reply:
@anastasiya29 But `node-headless-proxy` sample is experimental too.

Reply:
I haven't found a more "legalese" wording elsewhere. I emphasized "experimental" and I listed this sample in the Quick Start doc, under "Experimental Templates". If more need doing on that front, let me know.